src/test/ui/parser-recovery-4.rs

```diff
@@ -0,0 +1,21 @@
+// Copyright 2018 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+fn foo(_: usize) {}
+fn bar() {}
+
+fn main() {
+    let x = 1;
+    foo(x
+    bar()
```

Comment:
I need to modify the parser to not attempt to recover missing commas in the presence of missing braces to avoid E0061 here.